README.md

```diff
@@ -1,11 +1,13 @@
 ## Chainlink Local
 
-Chainlink Local is an installable package. It provides a simulation tool for developers to be able to depend on [Chainlink CCIP](https://docs.chain.link/ccip) locally in their Hardhat and Foundry projects.
+Chainlink Local is an installable dependency. It provides a tool (the Chainlink Local Simulator) that developers import into their Foundry and Hardhat scripts. This tool runs [Chainlink CCIP](https://docs.chain.link/ccip) locally which means developers can rapidly explore prototype and iterate CCIP dApps off-chain, and move to testnet only when they're ready to test in a live environment.
 
 The package exposes a set of smart contracts and scripts with which you build, deploy and execute CCIP token transfers and arbitrary messages on a local Hardhat or Anvil (Foundry) development node. Chainlink Local also supports forked nodes.
 
 User Contracts tested with Chainlink Local can be deployed to test networks without any modifications.
 
+> <b>NOTE: The use of Chainlink Local for CCIP development presumes you have prior knowledge of [Chainlink CCIP](https://docs.chain.link/ccip) and its primitives. We recommend going through the [tutorials](https://docs.chain.link/ccip/tutorials/) in the documentation prior to using this tool. It is also assumed that you are familiar with either [Harhdhat tooling](https://hardhat.org/) or [Foundry tooling](https://book.getfoundry.sh/).</b>
```

Comment:
this was a good point raised by @QingyangKong. We should note down as suggested that developers need to have some understanding of CCIP and its primitives before they will be able to successfully use the simulator